<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-3EB4RTA" failures="1" tests="2" name="Chek LoginErrorValidationMessage Of Products and Ordered Product" time="47.663" errors="0" timestamp="2024-08-25T12:13:22 IST">
  <testcase classname="Luma_Test.tests.Luma_ErrorValidation_Test" name="Luma_checkLoginError" time="2.603">
    <failure type="java.lang.AssertionError" message="expected [The account sign-in was incorrect or your account is disabled temporarily. Please wait and try again later.] but found [The account sign-in was or your account is disabled temporarily. Please wait and try again later.]">
      <![CDATA[java.lang.AssertionError: expected [The account sign-in was incorrect or your account is disabled temporarily. Please wait and try again later.] but found [The account sign-in was or your account is disabled temporarily. Please wait and try again later.]
at Luma_Test.tests.Luma_ErrorValidation_Test.Luma_checkLoginError(Luma_ErrorValidation_Test.java:39)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- Luma_checkLoginError -->
  <testcase classname="Luma_Test.tests.Luma_ErrorValidation_Test" name="productValidation" time="23.557"/>
</testsuite> <!-- Chek LoginErrorValidationMessage Of Products and Ordered Product -->
